src/saturn_engine/worker_manager/config/declarative_pipeline.py

```diff
@@ -0,0 +1,18 @@
+import dataclasses
+from typing import Any
+
+from saturn_engine.core.api import PipelineInfo as CorePipelineInfo
+
+
+@dataclasses.dataclass
+class PipelineSpec:
+    name: str
+    args: list[Any]
```

Comment:
I actually dropped `args` since this add uneeded complexity when parsing message and passing as arguments.

I suggest this new structure:

```
name: str
args: dict[str, object]
resources: dict[str, str]
```